$NetBSD: patch-ab,v 1.22 2015/05/09 03:58:15 mef Exp $

(1)
fix build on solaris 9
make opengl an option (on by default)
from PR#43047
http://gnats.netbsd.org/43047

(2) 
make: make: don't know how to make ca.po. Stop
Remove useless locale files that have no translations in them.
See post-extract: target in Makefile 

--- configure.in.orig	2014-10-09 03:09:41.000000000 +0900
+++ configure.in	2015-05-09 11:20:14.000000000 +0900
@@ -1245,7 +1245,7 @@ AC_GETTIMEOFDAY_ARGS
 AC_SYS_LARGEFILE
 AC_CHECK_FUNCS(select fcntl uname nice setpriority getcwd getwd putenv sbrk)
 AC_CHECK_FUNCS(sigaction syslog realpath setrlimit)
-AC_CHECK_FUNCS(setlocale)
+AC_CHECK_FUNCS(setlocale sqrtf)
 AC_CHECK_FUNCS(getaddrinfo)
 AC_CHECK_MEMBERS([struct sockaddr.sa_len],,, [[#include <sys/socket.h>]])
 AC_CHECK_ICMP
@@ -1294,7 +1294,7 @@ AC_DEFINE_UNQUOTED(PACKAGE, "$GETTEXT_PA
 		   generated code.])
 AC_SUBST(GETTEXT_PACKAGE)
 
-ALL_LINGUAS="ca da de es et fi fr hu it ja ko nb nl pl pt pt_BR ru sk sv vi wa zh_CN zh_TW"
+ALL_LINGUAS="da de es et fi fr hu it ja ko nb nl pl pt pt_BR ru sk sv vi wa zh_CN zh_TW"
 AM_GLIB_GNU_GETTEXT
 MKINSTALLDIRS="$INSTALL_DIRS"
 
